docs/guides/databases/postgresql/securely-manage-remote-postgresql-servers-with-pgadmin-on-macos-x/index.md

```diff
@@ -18,62 +18,76 @@ tags: ["database","postgresql"]
+## Install pgAdmin
 
-    A new window will be displayed in your web browser with the pgAdmin interface.
+1.  Visit the [pgAdmin download page](https://www.pgadmin.org/download/pgadmin-4-macos/) to obtain the most recent version. Save the installer to your desktop and launch it. Read the license agreement and click the "Agree" to continue.
 
-## Configure SSH Tunnel
+    ![pgAdmin on Mac OS X installer license agreement dialog](pg-admin-tos.png)
 
-While PostgreSQL supports SSL connections, it is not advisable to instruct it to listen on public IP addresses unless absolutely necessary. For this reason, you'll be using the following command to create an SSH tunnel to your database server, replacing `username` with your Linux username and `remote-host` with your Linode's hostname or IP address:
+1.  After the program is installed, you'll see a pgAdmin icon in a Finder window. You may drag this to your Applications folder or your dock.
 
-    ssh -f -L 5433:127.0.0.1:5432 username@remote-host -N
+1.  Start the pgAdmin interface. A welcome page should be displayed:
 
-Although PostgreSQL uses port 5432 for TCP connections, we're using the local port 5433 in case you decide to install PostgreSQL locally later on.
+    ![pgAdmin on Mac OS X menu bar icon menu](pg-admin-open-welcome-window.png)
 
 ## Use pgAdmin
 
-1.  Launch pgAdmin and you'll be presented with a default view containing no servers. Right click "Servers" and then navigate to "Create > Server".
-
-    ![pgAdmin III default view on Mac OS X](pg-admin-new-server.png)
-
-2.  If you're having problems connecting, you may need to check PostgreSQL's configuration to ensure it accepts connections. Modify the following lines in `/etc/postgresql/9.5/main/postgresql.conf` if necessary:
-
-    {{< file "/etc/postgresql/9.5/main/postgresql.conf" aconf >}}
-listen_addresses = 'localhost'
-
-port = 5432
-
-{{< /file >}}
+1. Open **pgAdmin 4**.
+2. In the **Quick Links** section, click **Add New Server**.
+3. Under the **General** tab, enter a name for your server connection. For example: `Linode PostgreSQL`
+4. Navigate to the **Connection** tab:
+   - **Hostname/address**: `localhost`.
+     The SSH tunnel redirects this to the Linode server.
+   - **Port**: The PostgreSQL port on your Linode, typically `5432`.
+   - **Maintenance Database**: `postgres` or your database name.
+   - **Username**: Your PostgreSQL username. For example: `postgres`
+   - **Password**: The password for your PostgreSQL user.
+5. Navigate to the **SSH Tunnel** tab:
+   - **Use SSH tunneling**: Enable this option.
+   - **Tunnel host**: Your Linode's IP address.
+   - **Tinnel port**: `22` . This is the default SSH port.
+   - **Username**: Your SSH username for the Linode instance.
+   - **Authentication**: Choose `Identity file` if you are using an SSH key, or `Password` for password-based authentication.
+   - **Identity file**: If you are using an SSH key, browse and select the private key file.
```

Comment:
@Rajakavitha1 is this accurate? I asks to select a "private key". Is this actually referring to a user's pubic key they would need to provide?

Reply:
Hi @jddocs  I checked the official documentation and this is what it reads: https://www.pgadmin.org/docs/pgadmin4/development/server_dialog.html
```Specify the type of authentication that will be used when connecting to the SSH host in the Authentication field:

Select the Password option to specify that pgAdmin will use a password for authentication to the SSH host. This is the default.

Select the Identity file to specify that pgAdmin will use a private key file when connecting.```

The users public key is uploaded when the hardening the [SSH access](https://techdocs.akamai.com/cloud-computing/docs/set-up-and-secure-a-compute-instance#harden-ssh-access) even before PostgreSQL is installed on Linode.  My understanding is that it is the private key that  is used to securely establish an SSH connection to the server. The server verifies this key against the public key stored in the ~/.ssh/authorized_keys file on the Linode instance.